Source/WPF/MyMoney/Database/Money.cs

```diff
@@ -12718,6 +12718,27 @@ public string InvestmentSecuritySymbol
             }
         }
 
+
+        [XmlIgnore]
+        public string SortableInvestmentSecurity
+        {
+            get
+            {
+                if (this.Investment != null && this.Investment.Security != null)
+                {
+                    if (String.IsNullOrEmpty(this.Investment.Security.Name))

```

Comment:
Hmmm, this doesn't seem to work for me.  Perhaps this line should instead be `if (!string.IsNullOrEmpty(this.Investment.Security.Symbol))` ?


![image](https://user-images.githubusercontent.com/3336415/228130856-558c4c17-9d75-4e38-a691-8703d8777607.png)

Reply:
You are correct. I pushed an update to this PR